examples/riot-kitchen-sink/src/stories/addon-actions.stories.js

```diff
@@ -0,0 +1,21 @@
+import { storiesOf } from '@storybook/riot';
+import { tag2, mount } from 'riot';
+import { action } from '@storybook/addon-actions';
+import ButtonRaw from './Button.txt';
+import { compileNow } from './compileNow';
+
+compileNow(tag2, ButtonRaw);
+
+storiesOf('Addon|Actions', module)
+  .add('Action only', () =>
+    mount('root', 'my-button', {
```

Comment:
Why support this mounting to 'root' in stories? It seems like an implementation detail to me. Would you like people to use it this way?

Reply:
I cannot get rid of this mount method (easily, unless if you have a better idea),
riot users are writing that statement to mount some elements on a particular node.

Users not wanting to mount on 'root' should use the other notations I provided.
Maybe we can even consider removing the mount notation if you prefer to.

Reply:
What are the cases when you prefer to mount on a particular node? I mean here in storybook you always want your component to be mounted to root. 
Maybe I don't understand the use case 🤷‍♂️

Reply:
Since I am not familiar with riot, I can't really argue about that =) 
I would say that having it in a public API is kinda ok, but strange. Also, it will complicate the ability to change the inner structure of Storybook without breaking users code.

If it's a rare case, I would reduce stories using it to the minimum - just to show the ability. If it's a common case - let's leave it as is.

Also, it's possible to expose some kind of helper function from storybook that will abstract it:

```js
import { stories, mount }  from '@storybook/riot';
```

Reply:
you want me to expose a dedicated version of mount ? it is worth the try, even if I am pretty sure that users will not want to use the impostering version

Reply:
Mmmh, I think I am wrong and you are right, it looks smoother to have mount and tag inside the @storybook/riot dependency.
Thanks for the idea

Reply:
Got rid of the 'root' anchor